src/components/FilterBar.jsx

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types';
 import { makeStyles, Grid } from '@material-ui/core';
 import FilterDropDown from './FilterDropdown';
 import AmountFilter from './AmountFilter';
+import {
+  QUERY_PARAM_MIN_AMOUNT,
+  QUERY_PARAM_MAX_AMOUNT,
+} from '../lib/QueryParams';
```

Comment:
I was thinking more like
```js
import qParams from '../lib/QueryParams';
```
And then using `qParams.MIN_AMOUNT` or something like that. This works too though. Also I'm not sure what the right name for the file should be so it's whatever.